website/www/site/content/en/documentation/basics.md

```diff
@@ -365,6 +373,76 @@ For more information about runners, see the following pages:
  * [Choosing a Runner](/documentation/#choosing-a-runner)
  * [Beam Capability Matrix](/documentation/runners/capability-matrix/)
 
+### Window
+
+Windowing subdivides a `PCollection` into _windows_ according to the timestamps
+of its individual elements. Windows enable grouping operations over unbounded
+collections by dividing the collection into windows of finite collections. A
+windowing function tells the runner how to assign elements to an initial window,
```

Comment:
one or more initial windows (maybe even zero is possible but that is wonky)

Reply:
You are 100% correct on all counts. The `WindowFn` takes an element as input and returns zero or more windows. The element is (conceptually) duplicated into each of those windows. So to be clear: in a PCollection each element is in precisely 1 window. But the function that says which window(s) to put it in can specify multiple, or none.

Reply:
Thanks for the clarification! I added a bit more text about this, and also in the programming guide for consistency. Seem accurate now?